internal/access/access.go

```diff
@@ -52,26 +54,6 @@ func RequireInfraRole(c *gin.Context, oneOfRoles ...string) (data.GormTxn, error
 			return db, nil
 		}
 	}
-
-	// check if they belong to a group that is authorized
-	groups, err := data.ListGroups(db, nil, data.ByGroupMember(identity.ID))
```

Comment:
I was able to remove this by using `IncludeInheritedFromGroups: true` in `Can` (below this).